src/index.html

```diff
@@ -5,10 +5,52 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="moyo-header">MOYO Header</h1>
+    <header class="header">
+      <a href="#image">
+        <img
+        class="image"
+        src="images\logo.png"
+        alt="logo">
+      </a>
```

Comment:
apply code styling recommendations from a checklist all over the page
```suggestion
      <a href="#image">
        <img
          class="image"
          src="images\logo.png"
          alt="logo"
        >
      </a>
```